CREATING_A_NEW_RELEASE.md

```diff
@@ -0,0 +1,24 @@
+# Instructions for creating a new release
+
+# Check everything locally:
+
+-   `npm run test`
+-   `npm run lint`
+
+# Preparing the code
+
+-   Bump version in package.json
```

Comment:
You could possibly also use `npm version patch` (or minor/major). Though this one is simple enough that bumping in package.json is possibly easier.